client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/form-content/email-notification.tsx

```diff
@@ -0,0 +1,77 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { ToggleControl } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import ConfigureEmailNotification from '../../configure-email-notification';
+import type { StateMonitorSettingsEmail } from '../../../sites-overview/types';
+
+interface Props {
+	recordEvent: ( action: string, params?: object ) => void;
+	verifiedItem?: { [ key: string ]: string };
+	enableEmailNotification: boolean;
+	setEnableEmailNotification: ( isEnabled: boolean ) => void;
+	defaultUserEmailAddresses: string[];
+	toggleAddEmailModal: () => void;
+	allEmailItems: StateMonitorSettingsEmail[];
+}
+
+export default function EmailNotification( {
+	recordEvent,
+	verifiedItem,
+	enableEmailNotification,
+	setEnableEmailNotification,
+	defaultUserEmailAddresses,
+	toggleAddEmailModal,
+	allEmailItems,
+}: Props ) {
+	const translate = useTranslate();
+
+	const isMultipleEmailEnabled: boolean = isEnabled(
+		'jetpack/pro-dashboard-monitor-multiple-email-recipients'
+	);
+
+	return (
+		<>
+			<div className="notification-settings__toggle-container">
+				<div className="notification-settings__toggle">
+					<ToggleControl
+						onChange={ ( isEnabled ) => {
+							recordEvent( isEnabled ? 'email_notification_enable' : 'email_notification_disable' );
+							setEnableEmailNotification( isEnabled );
+						} }
+						checked={ enableEmailNotification }
+					/>
+				</div>
+				<div className="notification-settings__toggle-content">
+					<div className="notification-settings__content-heading-with-beta">
```

Comment:
not blocking: I think we'll also need the beta flag for SMS, should this be broken out into a separate component now, or in a later PR.  I can always do a new PR after this is merged since I put the beta tag in the first place

Reply:
I think we can make this change when we start working on that project. I'm also ok with not breaking this into a separate component since this is a very small change.